src/Message/CIMCreateCardRequest.php

```diff
@@ -202,11 +202,13 @@ public function createPaymentProfile(CIMCreateCardResponse $createCardResponse)
 
         $getProfileResponse = $this->makeGetProfileRequest($parameters);
 
+        // Check if there is a pre-existing profile for the given card numbers.
+        $otherErrorCodes = array('E00039', 'E00042'); // For these codes we should check for duplicate payment profiles
```

Comment:
Yep, that's what I did. Lost track of how many zeroes were there in the error code.